py3status/modules/wifi.py

```diff
@@ -11,9 +11,8 @@
     device: Wireless device name (default "wlan0")
     down_color: Output color when disconnected, possible values:
         "good", "degraded", "bad" (default "bad")
-    format_down: Output when disconnected (default "W: down")
-    format_up: See placeholders below
-        (default "W: {bitrate} {signal_percent} {ssid}")
+    format: Display format for this module
+        (default 'W: [{bitrate} {signal_percent} {ssid}|\?show down]')
```

Comment:
>    I added W. Will remove ?show

this is not what I asked for.  I have been explicit.

> Btw, wifi deprecation does not work for me

what is/is not working?